plugin.js

```diff
@@ -0,0 +1,41 @@
+const cosmiconfig = require('cosmiconfig');
+
+function extractPluginName(pluginName) {
+    const atIndex = pluginName.lastIndexOf('@');
+    return atIndex !== -1
+        ? pluginName.substring(0, atIndex)
+        : pluginName;
+}
+
+async function findPluginConfig(pluginName) {
+    const { config } = await cosmiconfig.cosmiconfig('release').search();
```

Comment:
I feel `release` is too wide. I think we should look for only possible files, is possible using [searchPlaces](https://github.com/cosmiconfig/cosmiconfig?tab=readme-ov-file#searchplaces). All possibilities are mentioned in [semantic-release docs](https://semantic-release.gitbook.io/semantic-release/usage/configuration#configuration-file), and I hope it is possible to include `package.json` to enable the search for `package.json.release`

Reply:
This line should match the one in the [original project](https://github.com/semantic-release/semantic-release/blob/master/lib/get-config.js#L21) (`$GITHUB_WORKSPACE`) to get the correct configuration. Otherwise, we might end up with different configurations.

Reply:
Interesting. I wasn't aware of it. Thanks